### PR TITLE
chore: cfparams is changing default branch to main

### PR DESCRIPTION
**Pull request by therealvio**
# Purpose 🎯 

This change is part of an R&D experiment to see what it will take to switch the default branch from "master" to "main".

This PR contains changes for references to `master` in context to git branches.

# Context 🧠 

- [CSRE-2571](https://cultureamp.atlassian.net/browse/CSRE-2571)

[CSRE-2571]: https://cultureamp.atlassian.net/browse/CSRE-2571?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by therealvio** [comment]
⚠️ Before merging this branch, https://github.com/cultureamp/base-infrastructure-for-services/pull/3082 must be merged and deployed first.

File: .buildkite/hooks/pre-command [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env sh
 set -eou pipefail
 
-if [ "$BUILDKITE_BRANCH" = "master" ]; then
-  branch_type="master"
+if [ "$BUILDKITE_BRANCH" = "main" ]; then
+  branch_type="main"
```
Comment (jamestelfer) [inline .buildkite/hooks/pre-command:5]: The build role still includes `master` unless I'm mistaken

**Review by jamestelfer — CHANGES_REQUESTED** [review]
Check name of build role. I think you keep `master` for the assumed role. Check the use of `BRANCH_TYPE` elsewhere -- it may not be used.

File: .buildkite/hooks/pre-command (hunk 1 above)
Comment (therealvio) [reply .buildkite/hooks/pre-command:5]: If this is from `BIfS`, [my PR renamed it](https://github.com/cultureamp/base-infrastructure-for-services/pull/3082/files).

Comment (therealvio) [reply .buildkite/hooks/pre-command:5]: But that being said, I will check if that's the case now.

Comment (therealvio) [reply .buildkite/hooks/pre-command:5]: Ah, I see what you mean. By default the role is always going to be named `master`. Lemme fix that in a seperate commit for tracking reasons.

Comment (therealvio) [reply .buildkite/hooks/pre-command:5]: Also, in the repo, it doesn't appear that BRANCH_TYPE is used anywhere else in the repo.

**Review by jamestelfer — APPROVED** [review]

**Pull request merged**